platform/server/src/graphql.ts

```diff
@@ -18,19 +18,24 @@ export const createApolloServer = async function (app: any, config: Config) {
     const db = await connect(config);
     const pubSub = getPubSub();
 
-    const typeDefs = loadSchemaFiles(join(__dirname, '/schema/')).join('\n');
-    const schema = makeExecutableSchema({
-        typeDefs,
-        resolvers,
-        resolverValidationOptions: { requireResolversForResolveType: false },
-      });
-    const context = createMongoCRUDRuntimeContext(models, schema, db, pubSub);
+    const projectConfig = loadConfigSync({
+        extensions: [
+            () => ({ name: 'graphback' }),
+        ]
+    }).getDefault()
+
+    const graphbackConfig = projectConfig.extension('graphback');
+    const model = projectConfig.loadSchemaSync(path.resolve(graphbackConfig.model));
+    const runtimeEngine = new GraphbackRuntime(model, graphbackConfig);
+    const models = runtimeEngine.getDataSourceModels();
+    const context = createMongoCRUDRuntimeContext(models, model, db, pubSub);
```

Comment:
Will all of these be gone once we have the new runtime API? Seems like a lot of setup that a user should not have to care.

/cc @wtrocki @craicoverflow

Reply:
Yep, we are actively debating how to improve the new API

https://gist.github.com/craicoverflow/8c2631eda88771bff825700dad04bc3b#gistcomment-3313026

Feel free to weigh in with some ideas :)

Reply:
Cool. I'll check the gist.